examples/common/run-repart

```diff
@@ -9,7 +9,7 @@ chcon system_u:object_r:etc_t:s0 tmp/sysroot/state/*/etc/*
 
 definitions="${0%/*}/repart.d"
 
-if [ "${FIX_VERITY:-}" = '1' ]; then
+if [ "$SETUP_REPART" = '1' ]; then
```

Comment:
This variable is hilariously poorly named :rofl:

Reply:
It might be interesting to make this setup a bit more 'neutral' by dynamically generating the `repart.d` directory in the `tmp/` dir on each run using some sort of a heredoc-based approach... It's only two small files and it would be a bit cleaner than sed-hacking, I guess.